FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_IP.c

```diff
@@ -926,7 +926,7 @@ NetworkBufferDescriptor_t *pxResult;
 			/* The following statement may trigger a:
 			warning: cast increases required alignment of target type [-Wcast-align].
 			It has been confirmed though that the alignment is suitable. */
-			pxResult = * ( ipPOINTER_CAST( NetworkBufferDescriptor_t **, uxBuffer ) );
+			pxResult = * ( ( NetworkBufferDescriptor_t ** ) pucBuffer );
```

Comment:
This was correct, `uxBuffer` is converted to a pointer-pointer.

Reply:
I think we still have `uxBuffer` in place. I see this in the file diff [here](https://github.com/FreeRTOS/FreeRTOS/pull/246/files#diff-a9fb45b06179576c1ce6f497e7012d46R929). Can you please check and verify?

Reply:
In your reply there is a [link](https://github.com/FreeRTOS/FreeRTOS/pull/246/files#diff-a9fb45b06179576c1ce6f497e7012d46R929) that shows correctly:
~~~c
    pxResult = * ( ( NetworkBufferDescriptor_t ** ) uxBuffer );
~~~
I have no idea where the conflicts with master have come from.